testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -365,6 +382,31 @@ public void tupleTest()
         assertThat(tupleReturnValue.getInt(2)).isEqualTo(2);
     }
 
+    @Test
+	public void shouldFetchDataUsingPagination()
+	{
+		paginationTestSetup();
+
+		BoundStatement boundStatement =
+				session
+						.prepare(SimpleStatement.newInstance(
+								String.format("SELECT * FROM \"%s\".\"%s\" WHERE k = ?", keyspace, table)).setPageSize(15))
+						.bind(1);
+
+		// fetch the first page
+		ResultSet resultSet = session.execute(boundStatement);
```

Comment:
It would be nice to test both bound and simple statements because there are 2 distinct CQL execution paths, something like: https://github.com/stargate/stargate/commit/76f9e89eb79a86ef5afbb5c53594b764fa0af7cc

Reply:
Thank You for this suggestion, implemented in:
https://github.com/stargate/stargate/pull/66/commits/8023ea26700e806825b327d515538cd7ccc80ae2